vms/administrator/views.py

```diff
@@ -102,3 +103,43 @@ def post(self, request, *args, **kwargs):
 @admin_required
 def settings(request):
     return HttpResponseRedirect(reverse('event:list'))
+
+
+# View admin profile
+@login_required
+@admin_required
+def profile(request, admin_id):
+    admin = Administrator.objects.all().filter(id=admin_id)[0]
+    response = {"admin": admin}
+    return render(request, "administrator/profile.html", response)
+
+
+@login_required
+@admin_required
+def edit_profile(request, admin_id):
+    admin = get_administrator_by_id(admin_id)
+    organization_list = get_organizations_ordered_by_name()
+    if not admin:
+        return render(request, 'administrator/404.html')
+    else:
+        print request.POST
```

Comment:
why is this here ?

Reply:
I had it for debugging. I will change this. Thanks.